parsec/core/gui/password_validation.py

```diff
@@ -34,23 +36,36 @@ def get_password_strength_text(password_score):
     return PASSWORD_STRENGTH_TEXTS[password_score]
 
 
-def get_password_strength(password):
+def get_password_strength(password, user_inputs=None):
     if len(password) < 8:
         return 0
-    result = zxcvbn(password)
+    result = zxcvbn(password, user_inputs=user_inputs)
     return result["score"] + 1
 
 
 class PasswordStrengthWidget(QWidget, Ui_PasswordStrengthWidget):
     def __init__(self, parent=None):
         super().__init__(parent)
         self.setupUi(self)
+        self._user_inputs = []
+
+    @property
+    def user_inputs(self):
```

Comment:
`user_inputs` feels too vague for me (it took me a while to understand we were expecting to receive everything **but** the password)
We should rename this to something like `password_strength_context_strings`
On top of that it's unclear `user_inputs` is a list of strings, typing would be welcome here ;-) (another possibility would be to use `*user_inputs` as parameter so that it's obvious we receive a list of string)

And last, I'm not a big fan of using the @property setter (using a function makes it clear we are not hacking anything, and we don't have to provide a getter on the sanitized data that make expose it as a public attribute where it is supposed to be only internally used)

Reply:
Regarding my last rant, `PasswordChoiceWidget` also use `user_inputs` getter, however it could use a `PasswordStrengthWidget.get_password_strength(password)` method instead

Reply:
Renamed it to `excluded_strings`. It was named `user_inputs` because zxcvbn (the module used for password validation) calls it `user_inputs`.

Reply:
👍